intl/en/messages.json

```diff
@@ -0,0 +1,64 @@
+{
+
+    "Invalid usage (near option `{option}`), try {{`{program} --help`}}.":
```

Comment:
The messages.json looks quite interesting now; key === value.  Maybe we need to hash the keys in En before handoff.  Will think about it.